changelog.yaml

```diff
@@ -21,6 +21,15 @@
 - version: "NEXT"
   date: TBD
   changes:
+  - title: Introduce feature flags
+    type: internal
+    impact: patch
+    description: >-
```

Comment:
We need to change all `>-` to `|-`. Don't touch the existing ones to avoid conflicts.